server/src/routines/turn-actions/play-card.ts

```diff
@@ -23,7 +23,7 @@ function* playCardSaga(
 	const {playerId, rowIndex: pickedIndex, slot: pickedSlot} = pickInfo
 
 	const cardInfo = CARDS[card.cardId]
-	const {opponentPlayerId} = game
+	const opponentPlayerId = playerId === currentPlayer.id ? game.opponentPlayerId : currentPlayer.id
```

Comment:
Can you explain to me the reasoning behind this change and the consequences please?